libmscore/figuredbass.cpp

```diff
@@ -17,10 +17,13 @@
 #include "system.h"
 #include "segment.h"
 #include "chord.h"
+#include "staff.h"
 #include "rest.h"
 #include "score.h"
 #include "sym.h"
 #include "xml.h"
+#include "utils.h"
+#include "mscore/scoreview.h"
```

Comment:
This is not allowed. It breaks the fact that libmscore is a library not depending on  mscore.